Settings-default.toml

```diff
@@ -79,3 +79,7 @@ user_registration = true
 [odm]
 endpoint = "http://localhost:3000/"
 # TODO: authentication
+
+# Settings for compile-feature `ebv` only
+[ebv]
+api_base_url = "https://portal.geobon.org/api/v1"
```

Comment:
add new line at the end

Reply:
[c1d5cd6](https://github.com/geo-engine/geoengine/pull/493/commits/c1d5cd60d6942bb36c279414d9f1f071fbbbd6dc)